test/sequential/test-process-disable-warnings.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const execFile = require('child_process').execFile;
+const warnmod = require.resolve(fixtures.path('disable-warnings.js'));
+const node = process.execPath;
+
+const dep0025Message = /\(node:\d+\) \[DEP0025\] DeprecationWarning: sys is deprecated\. Use util instead\./;
+const dep0094Message = /\(node:\d+\) \[DEP0094\] DeprecationWarning: assert\.fail\(\) with more than one argument is deprecated\. Please use assert\.strictEqual\(\) instead or only pass a message\./;
+const experimentalWarningMessage = /\(node:\d+\) ExperimentalWarning: vm\.measureMemory is an experimental feature and might change at any time/;
+
+// Show All Warnings
+execFile(node, [warnmod], function(er, stdout, stderr) {
```

Comment:
Can you use `spawn` please? Also, consider using `common.spawnPromisified` if you prefer promises over callbacks